orocos_kdl/src/frames.cpp

```diff
@@ -87,10 +87,16 @@ namespace KDL {
 
     double Vector2::Norm() const
     {
-        if (fabs(data[0]) > fabs(data[1]) ) {
```

Comment:
Why not leaving the code below and add `if (data[0] == 0 && data[1]==0) return 0.0;` as first statement of the function?